chainer/functions/connection/convolution_nd.py

```diff
@@ -421,3 +421,254 @@ def convolution_nd(x, W, b=None, stride=1, pad=0, cover_all=False):
+    part of spatial locations. In this case, the output size is determined by
+    the following equations:
+
+    .. math::
+
+       l = (d + 2p - k + s - 1) / s + 1
+
+    Args:
+        x (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            Input variable of shape :math:`(n, c_I, d)`.
+        W (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            Weight variable of shape :math:`(c_O, c_I, k)`.
+        b (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            One-dimensional bias variable with length :math:`c_O` (optional).
+        stride (:class:`int`):
+            Stride of filter applications :math:`s`.
+        pad (:class:`int`):
+            Spatial padding width for input arrays :math:`p`.
+        cover_all (bool): If ``True``, all spatial locations are convoluted
+            into some output pixels. It may make the output size larger.
+            `cover_all` needs to be ``False`` if you want to use cuDNN.
+
+    Returns:
+        ~chainer.Variable:
+            Output variable of shape :math:`(n, l)`.
+
+    .. note::
+
+        This function uses cuDNN implementation for its forward and backward
+        computation if ALL of the following conditions are satisfied:
+
+        - ``cuda.cudnn_enabled`` is ``True``
+        - ``chainer.config.use_cudnn`` is ``'always'`` or ``'auto'``
+        - The number of spatial dimensions is more than one.
+        - ``cover_all`` is ``False``
+        - The input's ``dtype`` is equal to the filter weight's.
+        - The ``dtype`` is FP16, FP32 or FP64. (FP16 is only available when
+          cuDNN version :math:`\\geq` v3.)
+
+    Convolution links can use a feature of cuDNN called autotuning, which
+    selects the most efficient CNN algorithm for images of fixed-size,
+    can provide a significant performance boost for fixed neural nets.
+    To enable, set `chainer.using_config('autotune', True)`
+
+    .. seealso:: :class:`~chainer.links.ConvolutionND`
+
+    .. admonition:: Example
+
+        >>> n = 10
+        >>> c_i, c_o = 3, 1
+        >>> d = 50
+        >>> k = 10
+        >>> p = 5
+        >>> s = 6
+        >>> x = np.random.uniform(0, 1, (n, c_i, d)).astype('f')
+        >>> x.shape
+        (10, 3, 50)
+        >>> W = np.random.uniform(0, 1, (c_o, c_i, k)).astype('f')
+        >>> W.shape
+        (1, 3, 10)
+        >>> b = np.random.uniform(0, 1, (c_o,)).astype('f')
+        >>> b.shape
+        (1,)
+        >>> y = F.convolution_1d(x, W, b, stride=s, pad=p)
+        >>> y.shape
+        (10, 1, 9)
+        >>> l = int((d + 2 * p - k) / s + 1)
+        >>> y.shape == (n, c_o, l)
+        True
+        >>> y = F.convolution_1d(x, W, b, stride=s, pad=p, cover_all=True)
+        >>> y.shape == (n, c_o, l + 1)
+        True
+
+    """
+    func = ConvolutionND(1, stride, pad, cover_all)
```

Comment:
How about simply checking `x`'s `ndim` and calling `convolution_nd`?

Reply:
OK, will add a code for checking ndim